lib/membrane_rtc_engine/endpoints/hls/ffmpeg_filter.ex

```diff
@@ -1,84 +1,116 @@
-defmodule Membrane.RTC.Engine.Endpoint.HLS.FFmpegFilter do
-  @moduledoc """
-  Determines how compositor output will look be delivering ffmpeg filter graph.
-  """
+if Code.ensure_loaded?(VideoMixer) do
+  defmodule Membrane.RTC.Engine.Endpoint.HLS.FFmpegFilter do
+    @moduledoc """
+    Determines how compositor output will look be delivering ffmpeg filter graph.
+    """
 
-  @spec default_ffmpeg_filter(integer(), integer(), integer()) :: binary()
-  def default_ffmpeg_filter(width, height, inputs_num)
+    alias VideoMixer.FrameSpec
 
-  def default_ffmpeg_filter(width, height, 1) do
-    "[0:v]scale=#{width}:#{height}:force_original_aspect_ratio=decrease,setsar=1/1,pad=#{width}:#{height}:(ow-iw)/2:(oh-ih)/2"
-  end
+    @spec default_ffmpeg_filter(FrameSpec.t(), [FrameSpec.t()], any()) :: binary()
```

Comment:
Looks like this function could take `inputs_num` instead of doing `Enum.count` on its own

Reply:
I don't know how, because this function have to have the same type as `filter_graph_builder_t` https://github.com/kim-company/membrane_video_mixer_plugin/blob/1e16ee750425ef4d9fef154cf1ae0b852803b85c/lib/membrane/video_mixer/master_mixer.ex#L17-L19